src/Panel/panel.scss

```diff
@@ -54,42 +59,51 @@
         height: 100%;
         content: '';
         animation: loading 2s ease infinite;
-        background-color: $aquamarine;
+        background-color: $soft-loader-color;
       }
     }
   }
 
+  &.collapsable {
+    > .panel-header .panel-header-title-wrapper {
+      cursor: pointer;
+    }
+  }
+
   > .panel-header {
-    color: $white;
-    background-color: $charcoal;
+    color: $panel-header-color;
+    background-color: $panel-header-background-color;
```

Comment:
when using a `background-` variable please use always `background: $...`. This is needed in case we have to customize not only the color (for example when using a gradient)